src/algebra/order/floor.lean

```diff
@@ -24,7 +24,8 @@ We define the natural- and integer-valued floor and ceil functions on linearly o
 * `int.floor a`: Greatest integer `z` such that `z ≤ a`.
```

Comment:
You forgot to update l. 23.

Reply:
I don't quite understand this comment.

Reply:
Look at line 23. There's still "linear something" mentioned.